CMakeLists.txt

```diff
@@ -228,9 +228,11 @@ if(HAVE_ARM64_CRC32C)
     target_compile_options(crc32c_arm64 PRIVATE "-march=armv8-a+crc+crypto")
   endif(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
 endif(HAVE_ARM64_CRC32C)
+
 # CMake only enables PIC by default in SHARED and MODULE targets.
-set_property(TARGET crc32c_arm64
-    PROPERTY POSITION_INDEPENDENT_CODE "${BUILD_SHARED_LIBS}")
+if(BUILD_SHARED_LIBS)
+  set_property(TARGET crc32c_arm64 PROPERTY POSITION_INDEPENDENT_CODE "True")
```

Comment:
Please use TRUE instead of "True".

CMake's official documentation follows this style for POSITION_INDEPENDENT_CODE. https://cmake.org/cmake/help/latest/policy/CMP0083.html#policy:CMP0083

Reply:
Thanks for fixing the 'True'. I've never written cmake before, but I did spend 10 minutes trying to figure out if the value should be TRUE, ON, or 1. Now I know for future.

Reply:
I've pushed d89f780271969ceae904608d2c66999a6a98e38a to fix the style. Thanks.